python/lsst/pipe/tasks/ssoAssociation.py

```diff
@@ -186,7 +186,7 @@ def run(self, diaSourceCatalog, ssObjects, visitInfo, bbox, wcs):
                  'Obj_Sun_vz_LTC_km_s'],
                 ['ephRa', 'ephDec', 'phaseAngle', 'topoRange', 'helio_x', 'helio_y',
                  'helio_z', 'helio_vx', 'helio_vy', 'helio_vz'])
-            ssObjects['ssObjectId'] = [obj_id_to_ss_object_id(v) for v in ssObjects['ObjID']]
+            ssObjects['ssObjectId'] = [obj_id_to_ss_object_id(v) for v in ssObjects['packed_desig']]
```

Comment:
Looks good for now, though ultimately we'll want to rewrite `obj_id_to_ss_object_id` to take the unpacked designation and generate the ssObjectId. `adam_core` has routines to go from unpacked -> packed to maintain our clever scheme.